conda-forge.yml

```diff
@@ -1 +1,3 @@
 conda_forge_output_validation: true
+provider: {linux_aarch64: azure, linux_ppc64le: azure}
```

Comment:
isn't this supposed to be default instead of azure?

Reply:
oh maybe they changed it since travis has been doing badly recently...

Reply:
aarch runs on azure by default, and I hate travis for ppc as it breaks every other day. But happy to change if you prefer.

Reply:
I am not the maintainer, so my preference shouldn't matter :D 

I also hate travis!!! I am gonna copy this trick to other feedstocks

Reply:
THANK YOU, hopefully the osx runs pass too